layouts/partials/logo.html

```diff
@@ -1,2 +1,4 @@
-<img style="max-height:3em;display: inline-block" src="/images/icon_tight_crop.png" alt="RainbowCake logo"/>
-<a href="/">Rainbow​Cake</a>
+<a href="/">
+  <img style="max-height:3em;display: inline-block" src="/images/icon_tight_crop.png" alt="RainbowCake logo"/>
+</a>
+</a href="/">Rainbow​Cake</a>
```

Comment:
This opening `<a>` tag was broken on this line, though it will likely be removed anyway.